kolibri/auth/test/test_users.py

```diff
@@ -2,8 +2,7 @@
 
 from django.test import TestCase
 
-# Importing user models here results in a circular import... you should use get_user_model, but then there's no
-# way to get the proxy models as well. So just import at runtime.
+from kolibri.auth.models import FacilityUser, DeviceOwner, BaseUser, KolibriValidationError
```

Comment:
Nice, this is the stuff that wasn't working when the django env wasn't setup before running the tests?